tests/performance/src/test/java/org/mule/el/mvel/MVELDeepAssignPerformanceTestCase.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.el.mvel;
+
+import org.mule.DefaultMuleEvent;
+import org.mule.DefaultMuleMessage;
+import org.mule.MessageExchangePattern;
+import org.mule.api.MuleEvent;
+import org.mule.construct.Flow;
+import org.mule.tck.junit4.AbstractMuleContextTestCase;
+
+import java.util.Random;
+
+import org.databene.contiperf.PerfTest;
+import org.databene.contiperf.Required;
+import org.databene.contiperf.junit.ContiPerfRule;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class MVELDeepAssignPerformanceTestCase extends AbstractMuleContextTestCase
+{
+    @Rule
+    public ContiPerfRule rule = new ContiPerfRule();
+
+    @Override
+    public int getTestTimeoutSecs()
```

Comment:
Got that value from MVELPerformanceTestCase.
Should we reduce it in all these tests?